test.py

```diff
@@ -85,13 +85,14 @@ def test_user(self):
 
     def test_search(self):
         """test search result"""
-        result = self.loklak.search('doctor who')
+        result = self.loklak.search('doctor who', None, None, None, 18)
         self.assertTrue('error' in self.loklak.search())
         self.assertTrue('statuses' in result)
         self.assertTrue(isinstance(result['statuses'], list))
         self.assertTrue(len(result['statuses']) >= 1)
         self.assertEqual(len(result['statuses']),
                          int(result['search_metadata']['count']))
+        self.assertEqual(result['search_metadata']['itemsPerPage'], '18')
```

Comment:
The reason I used `itemsPerPage` instead of `count` was because in certain situations the `count` parameter did not match your input parameter. This could happened when there were not as many search results as you requested. In this case, the `itemsPerPage` parameter matches your input even if there are not enough results.